playbooks/bluetooth-pa-setup/bluetooth-pa-setup.yaml

```diff
@@ -0,0 +1,187 @@
+---
+- hosts: rpi
+  vars:
+    BLUETOOTH_NAME: NAME
+    DEPS: [ pulseaudio-module-bluetooth, git , python-dbus , libltdl-dev , pulseaudio , libtool , intltool , libsndfile-dev , libcap-dev , libjson-c-dev , libasound2-dev , libavahi-client-dev , libbluetooth-dev , libglib2.0-dev , autopoint , libsamplerate0-dev , libsbc-dev , libspeexdsp-dev , libssl-dev , libtdb-dev , libbluetooth-dev , intltool , autoconf , autogen , automake , build-essential , libasound2-dev , libflac-dev , libogg-dev , libtool , libvorbis-dev , pkg-config , python ]
+  tasks:
```

Comment:
Here you could write:
```
become: yes
tasks:
...
```
Instead of adding the `become: yes` to every single task.